clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAOSpec.scala

```diff
@@ -1,20 +1,22 @@
 package org.broadinstitute.clio.server.dataaccess
 
-import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
-  DocumentMock,
-  ElasticsearchIndex
-}
+import org.broadinstitute.clio.server.dataaccess.elasticsearch.{DocumentMock, ElasticsearchIndex}
 import org.broadinstitute.clio.server.dataaccess.elasticsearch.Elastic4sAutoDerivation._
-
 import com.sksamuel.elastic4s.circe._
 import io.circe.parser._
-import org.scalatest.{AsyncFlatSpec, Matchers}
-
+import org.scalatest.{AsyncFlatSpecLike, BeforeAndAfterAll, Matchers}
 import java.nio.file.Files
+import java.util.UUID
 
-class PersistenceDAOSpec extends AsyncFlatSpec with Matchers {
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.testkit.TestKit
+
+class PersistenceDAOSpec extends TestKit(ActorSystem("PeristenceDAOSpec")) with AsyncFlatSpecLike with BeforeAndAfterAll with Matchers {
```

Comment:
In need of a scalafmt-ing

Reply:
flatten works here ... but not elsewhere ... hm